src/materials/ADComputeISoilStress.C

```diff
@@ -523,32 +527,42 @@ ADComputeISoilStress<compute_stage>::computeStress()
       _strength_pressure_correction = 0.0;
   }
 
-  ADReal mean_pressure = 0.0;
+  mean_pressure = 0.0;
+
```

Comment:
Please correct the variable names in the .C file after making the changes in the .h file